LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Max Klein, Vivek Rai, Harsh Gupta
```

Comment:
`2014` -> `2015`
`Max Klein, Vivek Rai, Harsh Gupta` -> `WIGI Development Team`, you should rather mention our names in the README.